app/storage/storage_encryption.py

```diff
@@ -0,0 +1,66 @@
+import hashlib
+
+from flask import current_app
+
+from jwcrypto import jwe, jwk
+from jwcrypto.common import base64url_encode, base64url_decode
+from structlog import get_logger
+
+from app.utilities.strings import to_bytes
+from app.utilities.strings import to_str
+import simplejson as json
+
+logger = get_logger()
+
+
+class StorageEncryption:
+
+    def __init__(self, user_id, user_ik):
+        if user_id is None:
+            raise ValueError('User id must be set')
+        if user_ik is None:
+            raise ValueError('User ik must be set')
+
+        pepper = current_app.eq['secret_store'].get_secret_by_name('EQ_SERVER_SIDE_STORAGE_ENCRYPTION_USER_PEPPER')
```

Comment:
I don't think this class should access current_app, as this introduces the dependency of being in a flask app. Testing this also requires the app context whereas if pepper was passed in it could be tested in isolation.